Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -418,7 +418,7 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
         return true;
 
     RELEASE_LOG_ERROR(Extensions, "Access to this tab is not allowed for this extension");
-    RELEASE_ASSERT_NOT_REACHED();
```

Comment:
Should we keep a debug ASSERT_NOT_REACHED() so we can catch this if it continues to happen?

Reply:
yea i think that's a good idea. will add.